<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-11T11:35:57 IST" name="Suite" finished-at="2025-03-11T11:36:06 IST" duration-ms="9188">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.DemoNG@235834f2]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2025-03-11T11:35:57 IST" name="Test" finished-at="2025-03-11T11:36:06 IST" duration-ms="9188">
      <class name="TestRunner.DemoNG">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:TestRunner.DemoNG@235834f2]" started-at="2025-03-11T11:35:57 IST" name="setUpClass" finished-at="2025-03-11T11:35:57 IST" duration-ms="414" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@3b4a1a75]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.DemoNG@235834f2]" started-at="2025-03-11T11:35:57 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2025-03-11T11:36:06 IST" duration-ms="8722" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["To Check Page1 Login Funionality"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Unknown"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Ankit Singh] but found [manda user]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Ankit Singh] but found [manda user]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefinition.Scenario2.user_verifies_profile_account(Scenario2.java:33)
at âœ½.User verifies ProfileAccount(file:///C:/Users/Admin/eclipse-workspace/PageFactoryCode/src/test/resources/Feature/Sample3.feature:9)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:TestRunner.DemoNG@235834f2]" started-at="2025-03-11T11:36:06 IST" name="tearDownClass" finished-at="2025-03-11T11:36:06 IST" duration-ms="19" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- TestRunner.DemoNG -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
